packages/react/src/patterns/blocks/LeadSpaceBlock/LeadSpaceBlock.js

```diff
@@ -0,0 +1,106 @@
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { CTA } from '../../../components/CTA';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { ImageWithCaption } from '../../../components/ImageWithCaption';
+import { LinkList } from '../../sub-patterns/LinkList';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
```

Comment:
@greeshmareddyg Small thing, but we should change this import to be consistent with @asudoh's latest PR

```suggestion
import settings from 'carbon-components/es/globals/js/settings';
```

Reply:
@kennylam updated this.

Reply:
Thanks @greeshmareddyg @kennylam! 🙇